packages/app/src/app/pages/Sandbox/Editor/Workspace/items/NotOwnedSandboxInfo/NotOwnedSandboxInfo.tsx

```diff
@@ -35,8 +33,8 @@ export const NotOwnedSandboxInfo = () => {
       >
         <Dependencies />
       </WorkspaceItem>
-      {username === user.username && (
-        <TemplateInfo template={sandbox.customTemplate} />
+      {(currentSandbox.git || {}).username === user.username && (
```

Comment:
@SaraVieira Why not destructuring `currentSandbox` even further? 🤔

Reply:
I commented, we discussed this between me and ives and we find it more confuysing

Reply:
I wouldn't destructure `currentSandbox` too much, I find it personally hard to parse and you lose the info that `customTemplate` is defined on `currentSandbox`.

Reply:
I'm sorry, I didn't see the comment 😕

Maybe it's good to have a guideline about this somewhere? 🤔
Since now we places where everything is destructured really hard and other places (like this PR) don't.